homeassistant/components/xiaomi_miio/__init__.py

```diff
@@ -315,8 +311,6 @@ async def async_create_miio_device_and_coordinator(
         device = AirHumidifier(host, token, model=model)
         migrate = True
     # Airpurifiers and Airfresh
-    elif model == MODEL_AIRPURIFIER_3C:
-        device = AirPurifierMB4(host, token)
```

Comment:
For anyone reviewing this, this model was already listed in `MODELS_PURIFIER_MIOT` which provides the implementation.